### ISSUE TITLE
seed track URLs to MusicBrainz recordings

### ISSUE THREAD

**Issue by UltimateRiff** [kellnerd/harmony#37]
mostly for individual track pages, but I believe Bandcamp can have different licenses per track (tho I don't know if that'd be a recording or work URL...), for example

**Comment by kellnerd** [comment]
While this should not be difficult to implement, I imagine this feature would be a PITA to use with one tab for each recording being opened which has to be submitted.

**Comment by UltimateRiff** [comment]
this is true... I might propose a new MusicBrainz feature to make this easier? I feel like more relationships and data might be worth the work either way tho~

**Comment by kellnerd** [comment]
Since I already wrote the code to extract track IDs from URLs for #90, I've changed all providers to fill out recording/track IDs as well (3f6d834eeea91b41b7299b7987ba2601de43f5e7).

Now recording links are shown in the tracklist of looked up releases and I've also made it possible to seed them to MB (5e4d25c83cfdc4b705d76ed703c4f14e3bab3f0f). Since a release can have quite a lot of recordings, I've decided to put these actions last (after the cover images).
Personally I wouldn't recommend clicking all these links for releases with many tracks, but technically you can with today's update ;)

Maybe this will make the MBS ticket more likely to be implemented now as well as having a positive effect on BrainzPlayer's matching accuracy one day.
